src/app/backend/handler/apihandler.go

```diff
@@ -2330,3 +2320,11 @@ func parseNamespacePathParameter(request *restful.Request) *common.NamespaceQuer
 	}
 	return common.NewNamespaceQuery(nonEmptyNamespaces)
 }
+
+// getBaseParameter is used to get namespace, name, dataSelect in path parameter.
+func getBaseParameter(request *restful.Request) (string, string, *dataselect.DataSelectQuery){
```

Comment:
I don't think this is a good way to solve this. It would require a bigger refactoring to do it properly. We'd need some kind of struct/interface that would handle parameter extraction and expose it properly, so we could pass it further. I'd revert that for now.